### ISSUE TITLE
Enable more fuzz targets in oss-fuzz?

### ISSUE THREAD

**Issue by maflcko** [pauldreik/stdfuzz#8]
Currently only one is enabled in https://github.com/google/oss-fuzz/blob/b4407a1aa1998de44f13fa7ed0f93259cbc2ab8e/projects/libstdcpp/Dockerfile#L24 

It could make sense to enable the others as well, for example https://github.com/pauldreik/stdfuzz/blob/main/chrono/chrono-parse.cpp

**Comment by pauldreik** [comment]
Yes, I don't remember why, probably because they did not pass in gcc. Getting those chrono conversions right without overflows is tricky, it took some work in fmt.
